File: 2292-counting-words-with-a-given-prefix/2292-counting-words-with-a-given-prefix.js
/**
 * @param {string[]} words
 * @param {string} pref
 * @return {number}
 */
var prefixCount = function(words, pref) {
    if(words == null || words.length == 0) return 0;
    var cnt  = 0;
    for(let i = 0; i < words.length; i++){
        if(words[i].length >= pref.length && words[i].substring(0,pref.length) === pref){
            cnt++;
        }
    }

    return cnt;
};

File: 2292-counting-words-with-a-given-prefix/2292-counting-words-with-a-given-prefix.java
class Solution {
    public int prefixCount(String[] words, String pref) {
        if(words == null || words.length == 0){
            return 0;
        }

        int cnt  = 0;

        for(String s: words){
            if(s.length() >= pref.length() && s.substring(0,pref.length()).equals(pref)){
                cnt++;
            }
        }

        return cnt;
    }
}